tests/freqai/test_freqai_interface.py

```diff
@@ -19,7 +19,7 @@
 
 def is_arm() -> bool:
     machine = platform.machine()
-    return "arm" in machine or "aarch64" in machine
+    return "arm" in machine or "aarch74" in machine
```

Comment:
well this clearly has to be changed back - the architecture on an ARM is aarch64 - not aarch74.

Reply:
Whoops didn't mean to do that